pkg/bff/server_test.go

```diff
@@ -73,15 +73,25 @@ func (s *bffTestSuite) SetupSuite() {
 	// state doesn't change between tests.
 	go s.bff.Serve()
 
+	// Wait for 500 ms to ensure the BFF starts serving
+	time.Sleep(500 * time.Millisecond)
+
 	// Create the BFF client for making requests to the server
+	require.NotEmpty(s.bff.GetURL(), "no url to connect to client on")
```

Comment:
Was running into a race condition between `s.bff.Serve` and the `GetURL` function; added a sleep to prevent the race … which isn't great, but we don't have a "ready" signal.